build-scripts/bundle.js

```diff
@@ -66,7 +70,7 @@ module.exports.babelOptions = ({ latestBuild }) => ({
     ],
     // Only support the syntax, Webpack will handle it.
     "@babel/plugin-syntax-import-meta",
-    "@babel/syntax-dynamic-import",
+    "@babel/plugin-syntax-dynamic-import",
```

Comment:
We used to only load syntax because webpack dealt with the processing. What do we gain from adding this?

Reply:
Wasn't this just a typo?